base/bitarray.jl

```diff
@@ -828,6 +829,7 @@ function _resize_int!(B::BitVector, n::Int)
         Bc[end] = UInt64(0)
     end
     B.len = n
+    first && copy_chunks!(Bc, 1 + n - n0, Bc, 1, n0)
```

Comment:
This could be implemented more efficiently by `_growbeg!`ing Bc instead of growing at the end and copying.

Reply:
but how do you readjust the chunks after copying? I was inspired by `prepend!` which uses _growend!` as here`

Reply:
Ah, right. That optimization is only possible if the length increases by a multiple of 64.

Reply:
true, we could also apply it actually being so much faster than the default